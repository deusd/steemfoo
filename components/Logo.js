import React from 'react'
import { Text } from 'react-native'

export default props => {
  return <Text style={[styles.logo, props.style]}>Vybe</Text>
}

const styles = {
  logo: {
    fontFamily: 'MarckScript-Regular',
    color: 'white',
    fontSize: 72,
  },
}
