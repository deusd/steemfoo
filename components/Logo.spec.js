import 'react-native'
import { shallow } from 'enzyme'
import React from 'react'
import { Logo } from './Logo'

describe('Logo component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Logo />)
  })

  it('should render with no styles', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should render with styles', () => {
    wrapper = shallow(<Logo style={{ fontSize: 30 }} />)
    expect(wrapper.props().style).toContainEqual({ fontSize: 30 })
  })
})
