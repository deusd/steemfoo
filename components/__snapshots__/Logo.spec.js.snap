// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component should render with no styles 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "white",
        "fontFamily": "MarkScript",
        "fontSize": 72,
      },
      undefined,
    ]
  }
>
  Vybe
</Text>
`;
